apps/users/emails.py

```diff
@@ -104,6 +106,17 @@ def get_overview_link(self):
         link = self.get_host() + url
         return link
 
+    def get_manual_page_link(self):
+        site = Site.objects.filter(
+            is_default_site=True
+        ).first()
+        important_pages = ImportantPages.for_site(site)
+        if getattr(important_pages, 'platform_information') and \
+                getattr(important_pages, 'platform_information').live:
+            url = getattr(important_pages, 'platform_information').url
+            return self.get_host() + url
+        return None
```

Comment:
If using if/else in the template does not work, maybe we can return another link here? Like the landing page?
(And also if the if/else does not work, we cannot really use sendinblue for all emails, right? But that's another discussion and another point in time.)

Reply:
Yes, totally, we should return sth else then! Maybe the overview page? But let me try again with if/else in template first, I also was a bit impatient on Monday and should probably give it another go..

Reply:
@fuzzylogic2000 I still could not make if/else in template work :woman_shrugging: So, it now returns the overview link if manual page is not set.. what do you think?

Reply:
doing it this way has the advantage that it's all in one place I'd think, and not somewhere buried in a template at sendinblue which I will totally have forgotten and then will search forever :sweat_smile:

Reply:
Yeah, but we would like to not display the whole sentence when we don't have the link. But we also don't want to hand over the sentence as a variable. So, this with the fallback link is the best solution, I think!